src/rinterface.c

```diff
@@ -7449,11 +7449,7 @@ SEXP R_igraph_from_prufer(SEXP prufer) {
 
   SEXP r_result;
                                         /* Convert input */
-  if (!Rf_isNull(prufer)) {
-    R_SEXP_to_vector_int_copy(prufer, &c_prufer);
-  } else {
-    IGRAPH_R_CHECK(igraph_vector_int_init(&c_prufer, 0));
-  }
+  R_SEXP_to_vector_int_copy(prufer, &c_prufer);
```

Comment:
Is this change intended?

Reply:
Yes, this also needed

Reply:
Input argument in this function couldn't be `NULL`